packages/libs/sdk/package.json

```diff
@@ -30,10 +30,18 @@
     "@types/mocha": "^10.0.1",
     "@types/node": "^18.13.0",
     "@types/supertest": "^2.0.12",
+    "dotenv": "^16.0.3",
     "eslint-plugin-no-only-tests": "^3.1.0",
+    "graphql": "^16.6.0",
     "supertest": "^6.2.4"
   },
   "scripts": {
-    "codegen": "graphql-codegen --config codegen.ts"
+    "codegen": "graphql-codegen --require dotenv/config"
+  },
+  "exports": {
+    ".": {
+      "import": "./esm/index.js",
+      "require": "./cjs/index.js"
+    }
```

Comment:
Here is the part that dynamically will import the correct bundle based on whether the package is imported with `require` or `import`